### ISSUE TITLE
Ambiguous structure types in Oettingen

### ISSUE THREAD

**Issue by m-k94** [DCMLab/ddd#68]
While segmenting Oettingen, I often have the feeling that the editor creates some ambiguity between different types of text elements:

![image](https://user-images.githubusercontent.com/75686579/118474953-f2be1980-b70b-11eb-8056-55ee207ff658.png)

This looks like a graphic, but since the sentences connects the three lines it is is actually presented inline with the text. But it isn't fully that either, since the middle line is independent and everything has to be vertically aligned. 

another minor example from the previous page: 

![image](https://user-images.githubusercontent.com/75686579/118475041-0e292480-b70c-11eb-8844-824b885b596d.png)

this isn't important, but it shows the same sort of point: it's could be read as a simple text region, but then for some reason the word "Unterregnante" is seperated weirdly, with the second half being indented very far - making one think whether there is some graphical significance to that decision...

**Comment by fabianmoss** [comment]
Yes, there are many of these ambiguous cases in-between Figures and text paragraphs. As you note, the second case is less problematic and I agree with your solution to just have it as a separate paragraph (apparently, "Paralleltonart" was intended to be horizontally aligned). 

I think that it is completely OK to transcribe these instances as normal text, i.e. without line breaks and horizontal alignment.
